bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/EscalationValidator.java

```diff
@@ -51,6 +58,31 @@ private void validateEscalationCode(
     }
   }
 
+  private void validateIfReferredByCatchEvent(
+      final Escalation element, final ValidationResultCollector validationResultCollector) {
+    final Collection<CatchEvent> catchEvents = getAllElementsByType(element, CatchEvent.class);
+    final Optional<EventDefinition> catchEvent =
+        catchEvents.stream()
+            .flatMap(i -> i.getEventDefinitions().stream())
+            .filter(
+                e ->
+                    e instanceof EscalationEventDefinition
+                        && ((EscalationEventDefinition) e).getEscalation() == element)
+            .findFirst();
```

Comment:
❓ @remcowesterhoud. Why does this kind of scene exist? Can you give me an example?

Currently, in the same scope, We should only allow one handler to handle the escalation event. 🤔 

https://github.com/camunda/zeebe/blob/dc0727901b5ccdb891306927fe5f4484a6e0112c/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/util/ModelUtil.java#L285-L294

Reply:
It is validating the `Escalation`. Multiple catch events can reference the same `Escalation`. Scope is irrelevant here.

<img width="461" alt="image" src="https://user-images.githubusercontent.com/5787702/204503203-396655e1-5ad3-44e7-9cf4-87b0420facfe.png">
<img width="955" alt="Screenshot 2022-11-29 at 11 20 09" src="https://user-images.githubusercontent.com/5787702/204503738-407fd6cb-314f-40a5-87e9-36a7c00f88ef.png">

Reply:
This makes me think that it would be nicer to just move this validation to the `EscalationEventDefinitionValidator`. This is where we have this validation for error events as well. This will also make it more clear which element fails.